.azure-devops/graphitation-release.yml

```diff
@@ -41,3 +43,7 @@ jobs:
         displayName: Configure git for release
       - script: yarn release -y -n $(ossNpmToken) --access public
         displayName: Release
+      - template: ./steps/pierce-ado-npm-mirror-cache.yml
+        parameters:
+          adoNpmFeedPat: $(adoNpmFeedPat)
```

Comment:
Is this the correct way to make a secret available to a template?

Reply:
try just using "ossNpmToken" here.

Reply:
@kenotron So from our offline chat I assume you now understand that this isn't what we're after.

Reply:
I'll add this for you :)